.ci.yaml

```diff
@@ -26,7 +26,6 @@ platform_properties:
 targets:
   - name: Linux Cocoon
     builder: Linux Cocoon
-    postsubmit: false
```

Comment:
Do we want to run cocoon for post submit?

Reply:
Yes, it would be helpful to verify changes do not break the tree, and it's not very expensive to add a new linux vm test.